packages/button/src/MenuButton.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import styled from '@emotion/styled';
+import React, { ReactElement } from 'react';
+import { colors, spacingUnit, spacing } from '@ndla/core';
+import { Menu, MenuList, MenuItem, MenuButton as MenuButtonReach } from '@reach/menu-button';
+import { HorizontalMenu } from '@ndla/icons/contentType';
+import { useTranslation } from 'react-i18next';
+import { ButtonProps } from './';
+import { convertSizeForSVG } from './IconButton';
+
+interface StyledButtonProps {
+  svgSize: number;
+}
+
+const StyledMenuButton = styled(MenuButtonReach)<StyledButtonProps>`
+  cursor: pointer;
+  background-color: transparent;
+  border: none;
+  display: flex;
+  justify-content: center;
+  border-radius: 100%;
+  padding: ${({ svgSize }) => spacingUnit * (svgSize > spacingUnit ? 0.2 : 0.25)}px;
+  &:hover,
+  &:active,
+  &:focus {
+    background-color: ${colors.brand.light};
+    border: none;
+    box-shadow: none;
+  }
+  svg {
+    fill: ${colors.brand.secondary};
+    margin: 0;
+    width: ${({ svgSize }) => svgSize}px;
+    height: ${({ svgSize }) => svgSize}px;
+  }
+`;
+
+const StyledMenuList = styled(MenuList)`
+  border: none;
+  background-color: white;
+  padding: ${spacing.small};
+  border-radius: 4px;
+  box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.1), 0px 0px 20px rgba(0, 0, 0, 0.1);
+`;
+
+const StyledMenuItem = styled(MenuItem)`
+  gap: ${spacing.small};
+  padding: ${spacing.xxsmall};
+  display: flex;
+  cursor: pointer;
+  color: ${(prop) => prop.color};
+  span {
+    display: flex;
+    align-items: center;
+  }
+  svg {
+    fill: ${(prop) => prop.color};
+  }
+  :hover {
+    color: ${colors.brand.primary};
+  }
+`;
+
+interface MenuItemProps {
+  icon?: ReactElement;
+  text?: string;
+  onClick: () => void;
+  color?: string;
+}
+interface MenuButtonProps extends ButtonProps {
+  svgSize: StyledButtonProps;
```

Comment:
Må ikkje denne heite size for at dette skal gi meining? Ellers stemmer ikkje props med parameter til MenuButton.